### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by Skallwar**
- ld: Fix unhandled eh_frame
- ns16550: Remove QEMU_VIRT_BASE_ADDRESS constant from driver

### CODE REVIEW

File: boards/generic_qemuvirt.ld [hunk 1]
```diff
@@ -18,6 +18,10 @@ SECTIONS
         *(.rodata*);
     }
```
Comment (n1tram1) [inline boards/generic_qemuvirt.ld:20]: We should use this opportunity to also add .bss

File: src/drivers/ns16550.rs [hunk 2]
```diff
@@ -3,7 +3,6 @@
 
 use crate::kernel_console::Console;
 
-pub const QEMU_VIRT_BASE_ADDRESS: usize = 0x10000000;
 pub const QEMU_VIRT_NS16550_INTERRUPT_NUMBER: u16 = 10;
```
Comment (n1tram1) [inline src/drivers/ns16550.rs:6]: As we removed the `QEMU_VIRT_BASE_ADDRESS`, we should also remove `QEMU_VIRT_NS16550_INTERRUPT_NUMBER`

**Review by n1tram1 — CHANGES_REQUESTED** [review]

File: boards/generic_qemuvirt.ld (hunk 1 above)
Comment (Skallwar) [reply boards/generic_qemuvirt.ld:20]: Fixed

File: src/drivers/ns16550.rs (hunk 2 above)
Comment (Skallwar) [reply src/drivers/ns16550.rs]: Fixed

File: src/riscv64_qemuvirt_main.rs [hunk 3]
```diff
@@ -34,8 +33,15 @@ pub type InterruptsImpl = arch::riscv64::interrupts::Interrupts;
 pub type MemoryImpl = arch::riscv64::sv39::PageTable;
 pub type ConsoleImpl = Ns16550;
 
+pub const UART_ADDR: usize = 0x1000_0000;
```
Comment (n1tram1) [inline src/riscv64_qemuvirt_main.rs:36]: I liked the original qemu virt names

**Review by n1tram1 — APPROVED** [review]

File: src/riscv64_qemuvirt_main.rs (hunk 3 above)
Comment (Skallwar) [reply src/riscv64_qemuvirt_main.rs:36]: Since we are inside the QEMU main, what's the point of prefixing them?

**Pull request merged**